tests/Unit/ProvidesBrowserTest.php

```diff
@@ -31,34 +28,51 @@ public function test_capture_failures_for()
     }
 
     /**
-     * @dataProvider testData
+     * @dataProvider data
      */
-    public function test_store_console_logs_for()
+    public function test_capture_failures_for_data()
+    {
+        $browser = m::mock(stdClass::class);
+        $browser->shouldReceive('screenshot')->with(
+            'failure-Laravel_Dusk_Tests_Unit_ProvidesBrowserTest_test_capture_failures_for_data_foo-0'
+        );
+        $browsers = collect([$browser]);
+
+        $this->captureFailuresFor($browsers);
+    }
+
+    /**
+     * @dataProvider data
+     */
+    public function test_store_console_logs_for_data()
     {
         $browser = m::mock(stdClass::class);
         $browser->shouldReceive('storeConsoleLog')->with(
-            'Laravel_Dusk_Tests_Unit_ProvidesBrowserTest_test_store_console_logs_for-0'
+            'Laravel_Dusk_Tests_Unit_ProvidesBrowserTest_test_store_console_logs_for_data_foo-0'
         );
         $browsers = collect([$browser]);
 
         $this->storeConsoleLogsFor($browsers);
     }
 
-    public function test_truncate_test_name_where_that_name_is_too_long_and_might_cause_issues()
+    /**
+     * @dataProvider data
+     */
+    public function test_truncate_test_name_where_that_name_is_really_really_really_too_long_and_might_cause_issues_data()
     {
         $browser = m::mock(stdClass::class);
         $browser->shouldReceive('storeConsoleLog')->with(
-            'Laravel_Dusk_Tests_Unit_ProvidesBrowserTest_test_truncate_test_name_where_that_name_is_too_long_and_might_cause_is-0'
+            'Dusk_Tests_Unit_ProvidesBrowserTest_test_truncate_test_name_where_that_name_is_really_really_really_too_long_and_might_cause_issues_data_foo-0'
         );
         $browsers = collect([$browser]);
 
         $this->storeConsoleLogsFor($browsers);
     }
 
-    public static function testData()
+    public static function data()
```

Comment:
I took the `test` prefix off of this because PHPUnit is running the method as a test.